LucasTerra.lua

```diff
@@ -2512,37 +2512,54 @@ end
 -- @param	type	Type can be any alarm name you found on the alarms engine.
 -- @param	playsound	Set 'no' or 'yes' to play sound.
 -- @param	pausebot	Set 'no' or 'yes' to pause bot.
--- @param	logout	Set 'no' or 'yes' to logout.
+-- @param	logout		Set 'no' / 'Logout' / 'Force Logout' / 'Close Client' to logout.
+-- @param	cloudmsg	Set 'no' or 'yes' to send cloud message when alert is triggered.

```

Comment:
These docs should probably reflect that any truthy or falsy value can be used, instead of only `'yes'` and `'no'`.